### PR TITLE
Add compat data for SVG <path>

### PR DESCRIPTION
**Pull request by JeremiePat**
https://developer.mozilla.org/en-US/docs/Web/SVG/Element/path

### CODE REVIEW

File: svg/elements/path.json [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            "safari_ios": {
+              "version_added": "3.1"
+            }
+          },
+          "status": {
+            "experimental": false,
+            "standard_track": true,
+            "deprecated": false
+          }
+        },
```
Comment (SebastianZ) [inline svg/elements/path.json]: The `pathLength` attribute is missing.

SVG 1.1 also defined `externalResourcesRequired` and `transform` as attributes, which got removed in SVG 2. I'm not sure whether any browsers ever supported the former (or even still support them), but the latter is definitely still supported in browsers, see the examples at https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/transform. So, I guess we should include them.

File: svg/elements/path.json [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+                "version_added": "1"
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": true
+              },
+              "edge_mobile": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/path.json]: Apparently, Edge mobile has support for the `<path>` element, so it also must have support for the `d` attribute.

File: svg/elements/path.json [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+                "version_added": "4"
+              },
+              "ie": {
+                "version_added": "9"
+              },
+              "opera": {
+                "version_added": "8"
+              },
+              "opera_android": {
+                "version_added": null
```
Comment (SebastianZ) [inline svg/elements/path.json]: Apparently, Opera for Android has support for the `<path>` element, so it also must have support for the `d` attribute.

File: svg/elements/path.json (hunk 1 above)
Comment (JeremiePat) [reply svg/elements/path.json]: I haven't made up my mind about those two attributes. I'm considering provide compat information about `externalResourcesRequired` into the global attribute part as it is shared among many elements. For transform, I'm still wondering how to handle presentation attributes that can be used as CSS properties.

At that point I prefer not to add those two attributes and adding them back if it appears to be necessary.

**Comment by JeremiePat** [comment]
All fixed plus I revisit how to handle Safari version after reading those:

- https://en.wikipedia.org/wiki/Safari_version_history
- https://bugs.webkit.org/show_bug.cgi?id=5337

**Pull request merged**